dids/src/test/kotlin/web5/sdk/dids/methods/dht/DhtTest.kt

```diff
@@ -136,15 +136,17 @@ class DhtTest {
   inner class DhtTest {
     @Test
     fun `create and parse a bep44 put request`() {
+      print("WITCHCRAFT")
       val manager = InMemoryKeyManager()
-      val did = DidDht.create(manager)
+      val diddht = DidDhtApi {}
+      val did = diddht.create(manager)
 
       require(did.didDocument != null)
 
-      val kid = did.didDocument!!.verificationMethods?.first()?.publicKeyJwk?.get("kid")?.toString()
+      val kid = did.didDocument!!.verificationMethod?.first()?.publicKeyJwk?.keyID?.toString()
```

Comment:
just noticing casing on `keyID` here as opposed to `keyId`, does Kotlin have an idiomatic casing style?

Reply:
ah. that's from the nimbusdssd `JWK` library that provides that field with the casing. we are going to also roll our own JWK so i'll keep that in mind when i'm working on that issue